libs/components/src/lib/job-posting-second-form/job-posting-second-form.tsx

```diff
@@ -60,16 +60,16 @@ export function JobPostingSecondForm({ page }: IJobPostingFormProps) {
 		formState: { errors },
 	} = useForm<IJobPostingSecondForm>();
 
-	const array = selectDefaultArray(storedCountries, countries);
-
 	return (
 		<JobPostingGridForm id={page} onSubmit={handleSubmit(onSubmit)} justifyItems="center">
 			<IncreasedFieldWrapper gridRow={1} typeOfLength="full">
 				<JobPostingLabel>{countreisLabel}</JobPostingLabel>
 				<ErrorsHandlerWrapper positionRight={-20} width={15}>
 					<Controller
 						name="countries"
-						defaultValue={JSON.parse(JSON.stringify(array))}
+						defaultValue={JSON.parse(
+							JSON.stringify(selectDefaultArray(storedCountries, countries)),
+						)}
```

Comment:
I dont know what is it, but no, find other way

Reply:
removed JSON.parse, changed logic